lib/cf_obs_binary_builder/checksum.rb

```diff
@@ -48,6 +48,12 @@ class CfObsBinaryBuilder::Checksum
         "extension" => ".tar.gz"
       }
     },
+    "libgdiplus" => {
+      "type" => "github_tags",
+      "params" => {
+        "repo" => "mono/libgdiplus"
+      }
+    },
```

Comment:
This means depwatcher has support for verification.

Reply:
Not for libgdiplus:
```
{"source":{"name":"libgdiplus","type":"github_tags","repo":"mono/libgdiplus"},"version":{"ref":"5.6"}}
	
{"ref":"5.6","url":"https://github.com/mono/libgdiplus","git_commit_sha":"f07965ec5fc2b22bfb300d5fef410d25ae472a34"}
```

See [Trello card](https://trello.com/c/PWvmQNzj/583-adapt-cf-obs-binary-builder-to-get-rubygems-checksum-from-upstream)